src/datepicker/datepicker-month-view.ts

```diff
@@ -10,7 +10,7 @@ import {DayTemplateContext} from './datepicker-day-template-context';
   encapsulation: ViewEncapsulation.None,
   styleUrls: ['./datepicker-month-view.scss'],
   template: `
-    <div *ngIf="showWeekdays" class="ngb-dp-week ngb-dp-weekdays">
+    <div *ngIf="showWeekdays" class="ngb-dp-week ngb-dp-weekdays bg-light" role="row">
```

Comment:
Something is not right here. `bg-light` should not be here anymore. We removed it in c7bc4d7
 4 days ago.